jaas/src/main/java/com/griddynamics/jagger/jaas/controller/JaaSRestController.java

```diff
@@ -0,0 +1,78 @@
+package com.griddynamics.jagger.jaas.controller;
+
+import com.griddynamics.jagger.engine.e1.services.data.service.*;
+import io.swagger.annotations.ApiImplicitParam;
+import io.swagger.annotations.ApiImplicitParams;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * @author asokol
+ *         created 9/14/16
+ */
+interface JaaSRestController {
```

Comment:
general: make it clear what data is returned by every endpoint
can use docu for DataService for details